src/components/TreeView.js

```diff
@@ -34,6 +40,18 @@ export default class TreeView extends React.Component {
   }
 }
 
+function checkSignature(nodes, fetchedSigners) {
+  const signatureBuffer = Buffer.from(nodes[1].value.value, 'base64');
+  const sig = fetchedSigners.filter(x => x.sig.equals(signatureBuffer))[0];
```

Comment:
Better to use [`find`](https://lodash.com/docs/4.17.10#find) instead of `filter` here.